_component_design/tooltip.md

```diff
@@ -13,28 +13,292 @@ last-modified: 2017-08-17
+{: .hxSubSectionTitle}
+Tooltips have varying widths based on the amount of content, however, a tooltip relies on the following consistent spacing specifications:
+
+- 8px between vertical elements
+- 12px between unrelated elements 
+- max-width of 400px
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Spacing specifications for tooltip component."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-spacing-1.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Text values
+{: .hxSubSectionTitle}
+
+- 14px Roboto Medium for headers
+- 14px Roboto Regular for body copy
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Text specifications for tooltip component."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-spacing-2.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Pointer and placement
+{: .hxSubSectionTitle}
+
+- The pointer is 16 px wide at the base.
+- The apex is 8 px from the side of the rectangle. 
+- The pointer should start 12 px from the corner.
+- The side from which the pointer extends depends on the location of the trigger on the screen. Best practice pointer placement relies up the following:
+
+	- Always have the tooltip protrude toward the center of the screen.
+	- Ensure the tooltip does not hide important information.
+	- The tooltip content should be repositioned so that it doesn’t extend off the boundaries of the current screen.
+
+- The depth and width of the remaining side of the tooltip is dynamic based on how large the box needs to be with a max-width of 400px. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Variations of tooltip based on pointer placement."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-pointer-placement.svg){:width="100%"}
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+## Triggers
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Tooltips will only appear after be activated by a trigger. Triggers come in a variety of types.
 
--   Activates when hovered on click.
+### Trigger types
+{: .hxSubSectionTitle}
 
--   The tooltip disappears when mouse hovers away, or when clicked in the icon.
-    Tooltip does not reactivate on click.
+- **Icon:** Use this trigger variation if a recognizable icon on the application informs a user that additional information is available for consumption if needed. For example, A ‘?’ for help or ‘$’ for pricing.
```

Comment:
Last bit isn't a sentence. Also, use bold for examples of a GUI per style.
```markdown
for consumption if needed (for example, a **?** for help or a **$** for pricing).
```

Reply:
updated.